web/src/components/tasks/business-formation/business/WillPracticeLaw.tsx

```diff
@@ -1,27 +1,43 @@
+import { SnackbarAlert } from "@/components/njwds-extended/SnackbarAlert";
 import { FormationRadio } from "@/components/tasks/business-formation/business/FormationRadio";
 import { BusinessFormationContext } from "@/contexts/businessFormationContext";
 import { useConfig } from "@/lib/data-hooks/useConfig";
-import { ReactElement, useContext } from "react";
+import { ReactElement, useContext, useState } from "react";
 
 export const WillPracticeLaw = (): ReactElement => {
   const { Config } = useConfig();
   const { state, setFormationFormData, setFieldsInteracted } = useContext(BusinessFormationContext);
   const fieldName = "willPracticeLaw";
+  const [alert, setAlert] = useState<boolean>(false);
 
   return (
-    <FormationRadio
-      fieldName={fieldName}
-      title={Config.formation.fields.willPracticeLaw.label}
-      values={["true", "false"]}
-      onChange={(e): void => {
-        const valueToSave = JSON.parse(e.target.value);
-        setFormationFormData({
-          ...state.formationFormData,
-          [fieldName]: valueToSave,
-        });
-        setFieldsInteracted([fieldName]);
-      }}
-      errorMessage={Config.formation.fields.willPracticeLaw.error}
-    />
+    <>
+      <FormationRadio
+        fieldName={fieldName}
+        title={Config.formation.fields.willPracticeLaw.label}
+        subtitle={Config.formation.fields.willPracticeLaw.subtitle}
+        values={["true", "false"]}
+        onChange={(e): void => {
+          const valueToSave = JSON.parse(e.target.value);
+          const newFormationObject = {
+            ...state.formationFormData,
+            [fieldName]: valueToSave,
+          };
+          if (state.formationFormData.businessSuffix) {
+            newFormationObject.businessSuffix = undefined;
+            setFieldsInteracted(["businessSuffix"], { setToUninteracted: true });
```

Comment:
This is subject to change from an open question with the design team but it prevents the reseting to undefined from creating an error state on the BusinessSuffix dropdown.

Reply:
But its a one line change that I don't think interferes with the review.